src/main/java/org/sonarlint/intellij/core/SonarLintEngineManager.java

```diff
@@ -42,6 +44,13 @@
 
   public SonarLintEngineManager() {
     this(new SonarLintEngineFactory());
+    if (!ApplicationManager.getApplication().isUnitTestMode()) {
+      startSonarLintWebServer();
+    }
```

Comment:
Could we avoid using this if ? Maybe this is the sign that the start should not happen here, but in a different place that we would not reach from unit tests ?